packages/backend/src/routes/membership.route.ts

```diff
@@ -29,8 +28,7 @@ export const membershipRouter = (): IRouter => {
     membershipCtrl.businessValidation,
     membershipCtrl.post
   );
-  app.put('/:id', writePermission, membershipCtrl.put);
-  app.delete('/:id', writePermission, membershipCtrl.delete);
+  app.delete('/:id', writePermission, membershipCtrl.terminateMembership);
```

Comment:
terminate = delete ?

Reply:
No, because I think we are doing more here than deleting the membership here, so i wanted to explain thant by naming it differently than "delete"